backend/app/services/attachment/parser.py

```
@@ -981,16 +981,20 @@ def _parse_image(self, binary_data: bytes, extension: str) -> Tuple[str, str]:
-                if exif_data:
-                    # Just mention EXIF is available, don't extract all
-                    text += "\n包含 EXIF 元数据"
-
-            # Encode image to base64 for vision models
-            image_base64 = base64.b64encode(binary_data).decode("utf-8")
+            # Try to extract EXIF data if available (JPEG only; PNG raises OSError)
+            try:
+                if hasattr(img, "_getexif") and img._getexif():
+                    text += "\n\n[EXIF 信息]\n包含 EXIF 元数据"
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)